src/signpdf.test.js

```diff
@@ -147,6 +148,25 @@ describe('Test signing', () => {
         expect(typeof signature === 'string').toBe(true);
         expect(signedData instanceof Buffer).toBe(true);
     });
+    it('signs a ready pdf two times', async () => {
+        const p12Buffer = fs.readFileSync(`${__dirname}/../resources/certificate.p12`);
+        let pdfBuffer = fs.readFileSync(`${__dirname}/../resources/w3dummy.pdf`);
+        pdfBuffer = plainAddPlaceholder({
+            pdfBuffer, 
+            reason: 'first'
+        });
+        pdfBuffer = signer.sign(pdfBuffer, p12Buffer);
+        fs.writeFileSync(`${__dirname}/../resources/signed-once.pdf`, pdfBuffer);
```

Comment:
Writing to fs is OK while developing, but we don't want it in CI. So maybe just store a signed.pdf in resources and try to apply a sectond signature.
Ideally signatures would be verified with a devDependency (@MohammedEssehemy has the right thing for this), but that is a separate thing. Verifying however and whatever we can here is OK.

Reply:
We have made some progress in [verify-pdf](https://github.com/ninja-labs-tech/verify-pdf/) hopefully this will help

Reply:
Hey @MohammedEssehemy ! I tried to use verify-pdf but when I ran it over a pdf with two signatures I get this output:
``` 
    console.log src/signpdf.test.js:162
      { verified: false,
        authenticity: false,
        integrity: true,
        expired: false,
        meta:
         { certs: [ [Object] ],
           signatureMeta:
            { reason: 'first',
              contactInfo: 'emailfromp1289@gmail.com',
              location: 'Location from p12' } } }
```
It doesn't seem to recognize the second signature.
If I run mutools I get this output:
```
➜  node-signpdf git:(develop) ✗ mutool sign -v resources/signed-twice.pdf                   
verifying signature 18
  Designated name: CN=signpdf, O=Some Company, OU=, emailAddress=signpdf@example.com, C=BG
  Certificate error: Self-signed certificate.
  The signature is valid but there have been edits since signing.
verifying signature 21
  Designated name: CN=SignPdf, O=node-signpdf, OU=, emailAddress=node-signpdf@example.com, C=BG
  Certificate error: Self-signed certificate.
  The document is unchanged since signing.
```
The file is this:
[signed-twice.pdf](https://github.com/vbuch/node-signpdf/files/3810645/signed-twice.pdf)

Reply:
We are still working on adding support for multi signature @andres-blanco

Reply:
Either way, this is a "good to have" but not a mandatory check. As long as we do some testing.

Reply:
I've created an issue here
https://github.com/ninja-labs-tech/verify-pdf/issues/8
and will try working on it, once we have try free time.